Makefile

```diff
@@ -637,7 +637,7 @@ release-hyperkit-driver: install-hyperkit-driver checksum ## Copy hyperkit using
 
 .PHONY: check-release
 check-release: ## Execute go test
-	go test -v ./deploy/minikube/release_sanity_test.go -tags=release
+	go test -timeout 42m -v ./deploy/minikube/release_sanity_test.go
```

Comment:
why is the timeout changed to 42minutes?

Reply:
The test downloads all the releases, it takes some time. Now we're going to check beta releases too and it's going to be event longer. The default timeout was 10 minutes and it wasn't enough.